core/src/main/java/tc/oc/pgm/controlpoint/ControlPointParser.java

```diff
@@ -77,11 +77,36 @@ public static ControlPointDefinition parseControlPoint(
     Duration timeToCapture =
         XMLUtils.parseDuration(elControlPoint.getAttribute("capture-time"), Duration.ofSeconds(30));
 
+    final double decayRate, recoveryRate, ownedDecayRate;
+    final Node attrIncremental = Node.fromAttr(elControlPoint, "incremental");
+    final Node attrDecay = Node.fromAttr(elControlPoint, "decay-rate");
+    final Node attrRecovery = Node.fromAttr(elControlPoint, "recovery-rate");
+    final Node attrOwnedDecay = Node.fromAttr(elControlPoint, "owned-decay-rate");
+    if (attrIncremental == null) {
+      recoveryRate =
+          XMLUtils.parseNumber(attrRecovery, Double.class, koth ? 1D : Double.POSITIVE_INFINITY);
+      decayRate =
+          XMLUtils.parseNumber(attrDecay, Double.class, koth ? 0.0 : Double.POSITIVE_INFINITY);
+      ownedDecayRate = XMLUtils.parseNumber(attrOwnedDecay, Double.class, 0.0);
+    } else {
+      if (attrDecay != null)
+        throw new InvalidXMLException("Cannot combine this attribute with incremental", attrDecay);
+      if (attrRecovery != null)
+        throw new InvalidXMLException(
+            "Cannot combine this attribute with incremental", attrRecovery);
+      if (attrOwnedDecay != null)
+        throw new InvalidXMLException(
+            "Cannot combine this attribute with incremental", attrOwnedDecay);
+
+      final boolean incremental = XMLUtils.parseBoolean(attrIncremental, koth);
+      recoveryRate = incremental ? 1.0 : Double.POSITIVE_INFINITY;
+      ownedDecayRate = 0.0;
+      decayRate = incremental ? 0.0 : Double.POSITIVE_INFINITY;
```

Comment:
```suggestion
      recoveryRate = incremental ? 1.0 : Double.POSITIVE_INFINITY;
      decayRate = incremental ? 0.0 : Double.POSITIVE_INFINITY;
      ownedDecayRate = 0.0;
```
For consistency, keep the order of the 3